ssl/ssl_cert.cc

```diff
@@ -274,7 +274,7 @@ static enum leaf_cert_and_privkey_result_t check_leaf_cert_and_privkey(
 }
 
 static int cert_set_chain_and_key(
-    CERT *cert, CRYPTO_BUFFER *const *certs, size_t num_certs,
+    CERT *cert, UniquePtr<STACK_OF(CRYPTO_BUFFER)> *certs, size_t num_certs,
```

Comment:
```suggestion
    CERT *cert, UniquePtr<STACK_OF(CRYPTO_BUFFER)> &certs, size_t num_certs,
```

Since it's C++ pass by reference rather then passing the address. You'll have to fix up the other spots where this is called then.

Reply:
Also assumes this works with the usage of `std::move`?

Reply:
Made the change, based on some google searching std::move should work on references to UniqPtrs.